x-pack/plugins/ml/public/maps/layer_selector.tsx

```diff
@@ -58,7 +58,7 @@ export class LayerSelector extends Component<Props, State> {
           options={[
             { value: 'actual', label: 'actual' },
             { value: 'typical', label: 'typical' },
-            { value: 'connected', label: 'connected' },
+            { value: 'typical to actual', label: 'typical to actual' },
```

Comment:
Should the labels here be internationalized?

Reply:
Good catch! Added in 2321384960d2018fc3aa3e54ddf0d4e5c95ae80a